core/data-api/src/main/java/com/dev/innerview/core/data_api/InnerViewRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.dev.innerview.core.data_api
```

Comment:
🚫 **[ktlint]** <standard:package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore
<!-- __reviewdog__:ChA5NDA4ODIwZmRkMWFkNTBkEgZrdGxpbnQ= -->